packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1327,6 +1327,7 @@ export class Profiles extends ProfilesCache {
 
     private async updateBaseProfileFileLogin(profile: IProfileLoaded, updProfile: IProfile) {
         const upd = { profileName: profile.name, profileType: profile.type };
+        // const config = (await this.getProfileInfo()).getTeamConfig();
```

Comment:
Just a minor thing ; better to not have commented out codes.
Is there a specific reason for it ?

Reply:
I agree about no commented-out code :yum:
The only reason (I believe) I left it there was to serve as a reminder right after the release goes out.
That way, when we bump the CLI (and imperative) version, we remember to remove any `ImperativeConfig.instance.config`.

Is it ok to leave this one in ?
I will open a PR to address this, right after opening the Release PRs. 😋